README.md

```diff
@@ -378,7 +378,7 @@ method takes one argument:
 
 1. `b` must be an Apply of a function
 
-    1. If `b` does not represent a function, the behaviour of `ap` is
+    1. If `b` does not represent an Apply of a function, the behaviour of `ap` is
        unspecified.
```

Comment:
I think saying that `b` must be **an Apply of** a function is double, as we already stated that right before:

> 1. `b` must be **an Apply of** a function

I read *represent* to mean *the value represented by the Apply*, or *the inner value*. So in my opinion, the original sentence was alright.